operator/validator/controller.go

```diff
@@ -452,21 +444,11 @@ func (c *controller) StartValidators() {
 		return
 	}
 
-	intBuf := new(big.Int)
-
-	mySubnets := c.selfSubnets(intBuf)
-
 	var ownShares []*ssvtypes.SSVShare
-	var pubKeysToFetch [][]byte
 	for _, share := range shares {
 		if c.operatorDataStore.GetOperatorID() != 0 && share.BelongsToOperator(c.operatorDataStore.GetOperatorID()) {
 			ownShares = append(ownShares, share)
 		}
-		networkcommons.SetCommitteeSubnet(intBuf, share.CommitteeID())
-		subnet := intBuf.Uint64()
-		if mySubnets[subnet] != 0 {
-			pubKeysToFetch = append(pubKeysToFetch, share.ValidatorPubKey[:])
-		}
```

Comment:
I can't find where this part is re-added?

Reply:
added in https://github.com/ssvlabs/ssv/pull/1805/commits/8a742345cfe887311403c65429f03c5b0cadc45a

Reply:
and https://github.com/ssvlabs/ssv/pull/1805/commits/db78db1b8f54e864b28e72b1e7848108bf99cbe3